TestAppTypescript/src/CrashesScreen.tsx

```diff
@@ -60,7 +60,7 @@ export default class CrashesScreen extends Component<
       component.setState({ lastSessionStatus: status });
     }
 
-    const textAttachmentValue = await AttachmentsProvider.getTextAttachment();
+    const textAttachmentValue = await AttachmentsProvider.getTextAttachment() || "";
```

Comment:
Isn't `getTextAttachment()`(https://github.com/microsoft/appcenter-sdk-react-native/blob/develop/TestAppTypescript/src/AttachmentsProvider.ts#L68) already returning empty string when it can't get a text attachment? Why do we need empty string here?

Reply:
As `AsyncStorage.getItem` has `Promise<string | null>` return type, typescript compiler returns an error:
`Type 'string | null' is not assignable to type 'string’.`
I removed that line and updated `AttachmentsProvider.getItemFromStorage` method to handle this type check. Please, check it out.